app/src/main/java/net/gsantner/markor/ui/NewFileDialog.java

```diff
@@ -171,6 +171,11 @@ private AlertDialog.Builder makeDialog(final File basedir, final boolean allowCr
                     final byte[] templateContents = getTemplateContent(templateSpinner, basedir, f.getName(), encryptCheckbox.isChecked());
                     shareUtil.writeFile(f, false, (arg_ok, arg_fos) -> {
                         try {
+                            if (appSettings.isEditorAddBomToNewFile()) {
```

Comment:
Maybe just make it a toggle on the newfile dialog, not a general settings option. so when you create a new file you tick that on or off if you want to.

what do you think?

Reply:
That's a good idea.